android/CouchbaseLite/build.gradle

```diff
@@ -31,6 +31,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.0'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
+        compileOnly 'com.android.support:support-annotations:28.0.0'
```

Comment:
That is wrong location. Please put it in the dependencies section around line number 132.